src/models/pang12/repairs.md

```diff
@@ -0,0 +1,174 @@
+# Pangolin (Parts & Repairs)
+
+Many components in your Pangolin can be upgraded or replaced as necessary. Follow these step-by-step guides for instructions:
+
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Removing the battery](#removing-the-battery)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the speakers](#replacing-the-speakers)
+
+## Replacing an M.2/NVMe SSD:
+
+This model supports two M.2 SSDs. Both slots support PCIe NVMe Generation 4 connections and M.2 size 2280.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 3 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the M.2 drive:
+
+1. Unscrew the M.2 access door screw.
+
+![M.2 access door](./img/m2-access-door.webp)
+
+2. Pull the M.2 access door off, starting from the top (larger side).
+    - A flat, plastic tool can be used to pry the edge of the door up.
+    - If the door is difficult to remove, the M.2 drives can alternatively be accessed by [removing the bottom cover](#removing-the-bottom-cover).
+3. Unscrew the retainer screw opposite the M.2 slot.
+
+![M.2 slots](./img/m2-slots.webp)
+
+4. Remove the existing M.2 drive by pulling it out of the slot.
+5. Insert the new M.2 drive into the slot and hold it in place.
+    - If a slot is being used for the first time, the spare thermal pad (which is included in the Pangolin's box) can be applied to the motherboard before placing the drive down on top of it.
```

Comment:
The thermal pad doesn't really go on the motherboard here, since the M.2 connectors are at the very edge of the board. It goes on the little silver heatsink thing, however you want to describe that.

Reply:
[8fa492a](https://github.com/system76/tech-docs/pull/164/commits/8fa492a9a24f64c5bc89f2bb00bd9f4aaa6a0c13) - `silver backplate`, how's that?

Reply:
Sounds great!